src/pages/ProfilePage/views/AccountSettingsView/AccountSettingsView.tsx

```diff
@@ -51,24 +67,28 @@ function AccountSettingsView() {
     resolver: yupResolver(profilePageSchema),
   });
 
+  useEffect(() => {
+    setValue('name', user.name);
+    setValue('email', user.email);
+    setImg(user.avatar);
+  }, [setValue, user.avatar, user.email, user.name]);
+
   const avatarUrl = () => {
-    if (formData?.image && typeof formData?.image === 'string') {
-      return formData.image;
+    if (img && typeof img === 'string') {
+      return `http://localhost:4000/${user.avatar}`;
```

Comment:
`http://localhost:4000` would be best made a variable `HOST_URL` and kept inside `.env` so that the deployed app and a local one have their own versions of this.